### PR TITLE
[Refactor]: added primary and second color as token

### PR DESCRIPTION
**Pull request by Rajat-Dabade**
Primary and second color tokens.

### CODE REVIEW

File: packages/token/tokens/color/base.json [hunk 1]
```diff
@@ -2,13 +2,15 @@
   "color": {
     "base": {
       "gray": {
-        "light" : { "value": "#CCCCCC", "darkValue": "#EDEDED" },
-        "medium": { "value": "#999999" },
-        "dark"  : { "value": "#111111" }
+        "light": {
+          "value": "rgba(255, 255, 255, 0.45)"
```
Comment (palashgdev) [inline packages/token/tokens/color/base.json:6]: color is independent of opacity can't we use hex here or color without opacity

File: packages/token/tokens/color/base.json [hunk 2]
```diff
@@ -2,13 +2,15 @@
...
+        }
       },
-      "red": { "value": "#FF0000" },
-      "green": { "value": "#00FF00" },
-      "navyblue": { "value": "#1668DC" }
+      "navyblue": {
+        "light": {
+          "value": "rgba(22, 104, 220, 1)"
```
Comment (palashgdev) [inline packages/token/tokens/color/base.json:11]: same here

File: packages/token/tokens/size/font.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
   "size": {
     "font": {
       "normal": {
```
Comment (palashgdev) [inline packages/token/tokens/size/font.json]: can we make this regular

Comment (Rajat-Dabade) [reply packages/token/tokens/size/font.json]: Okay.

File: packages/token/tokens/color/base.json (hunk 1 above)
Comment (Rajat-Dabade) [reply packages/token/tokens/color/base.json:6]: Screenshot 2023-09-26 at 12 39 54 PM

There are some colour tokens with opacity included in the Figma.

**Pull request merged**